infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/V2APIConstants.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.service.smartevents.infra.v2.api;
+
+import com.redhat.service.smartevents.infra.core.api.APIConstants;
+
+/**
+ * Some constants used for URL construction for the API
+ */
+public class V2APIConstants {
+
+    /**
+     * Base Path for the Smart Events application API v2.
+     */
+    public static final String V2_ROOT = APIConstants.ROOT + "/v2";
+
+    /**
+     * Base Path for the Cloud Providers API
+     */
+    public static final String V1_CLOUD_PROVIDERS_BASE_PATH = V2_ROOT + "/cloud_providers";
```

Comment:
```suggestion
    public static final String V2_CLOUD_PROVIDERS_BASE_PATH = V2_ROOT + "/cloud_providers";
```
Same with other constants below